build/common.props

```diff
@@ -13,7 +13,7 @@
     <Company>Microsoft</Company>
     <Product>Azure WebJobs SDK Script Runtime</Product>
     <PackageProjectUrl>http://go.microsoft.com/fwlink/?LinkID=320972</PackageProjectUrl>
-    <RepositoryUrl>https://github.com/Azure/azure-webjobs-sdk-script</RepositoryUrl>
+    <RepositoryUrl>https://github.com/Azure/azure-functions-host.git</RepositoryUrl>
```

Comment:
nit: can we drop the `.git` to be consistent with the rest of the packages (WebJobs, Functions, ASP.NET Core)?